twoliter/embedded/Makefile.toml

```diff
@@ -850,8 +851,15 @@ export PATH="${TWOLITER_TOOLS_DIR}:${PATH}"
 # Parse the variant into its components and set additional variables.
 eval "$(bottlerocket-variant)"
 
+OUTPUT_LOGS_DIR="${BUILDSYS_LOGS_DIR}/${BUILDSYS_ARCH}-${BUILDSYS_VARIANT}"
+REPACK_OUTPUT_LOG="${OUTPUT_LOGS_DIR}/${BUILDSYS_VERSION_FULL}/repack.out"
+
+rm -rf "${OUTPUT_LOGS_DIR:?}"
+mkdir -p "${OUTPUT_LOGS_DIR}/${BUILDSYS_VERSION_FULL}"
 buildsys repack-variant \
-  --cargo-manifest-dir variants/${BUILDSYS_VARIANT}
+  --cargo-manifest-dir "variants/${BUILDSYS_VARIANT}" \
+  > "${REPACK_OUTPUT_LOG}" || cat "${REPACK_OUTPUT_LOG}"
```

Comment:
this shows the log on error but clobbers the non-zero exit code, which we need to pass up to avoid reporting success